File: BOJ(Java 공부용)/10871.java
import java.io.*;
public class Main {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        String arr[] = br.readLine().split(" ");
        int n= Integer.parseInt(arr[0]),x = Integer.parseInt(arr[1]);
        String s[] = br.readLine().split(" ");
        int sum =0;
        for(int i=0;i<s.length;i++) {
            int num = Integer.parseInt(s[i]);
            if(num<x) {
                bw.write(String.valueOf(num)+" ");
                bw.flush();
            }
        }
    }
}

File: BOJ(Kotlin)/10871.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter
import java.util.*

fun main(args: Array<String>) {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val bw = BufferedWriter(OutputStreamWriter(System.out))

    val st = StringTokenizer(br.readLine())
    val n = st.nextToken().toInt()
    val x = st.nextToken().toInt()
    val StringList : List<String> = br.readLine().split(" ")
    for (value in StringList) {
        val item = value.toInt()
        if (item < x.toInt()) {
            bw.write("$item ")
        }
    }
    bw.flush()
    bw.close()
}